src/org/folio/Constants.groovy

```diff
@@ -148,6 +148,8 @@ class Constants {
     // List of module for CONSORTIUM_ENABLED=true
     static List CONSORTIUM_ENABLED = ["mod-search"]
 
+    static List CROSS_TENANT_ENABLED = ["mod-authtoken"]
```

Comment:
This property only required only by this module then this constant is not required

Reply:
Thanks Sasha!